packages/react-sync-with-portabl/src/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react';
+import { createSyncWithPortabl, Options } from '@portabl/js-sync-with-portabl';
+
+const SyncWithPortabl = ({
+  providerName,
+  envOverride,
+  clientId,
+  rootSelector,
```

Comment:
I think we can remove this as a prop and return the root selector instead of null. So that it will render where they set the component.